### DISCUSSION TITLE
Fermata Auto. Continue watching the video after closing the app.

### DISCUSSION THREAD

**Discussion by picozzo71** [AndreyPavlenko/Fermata#282] [General]
Hi, after closing the stopped app fermata, is it possible to take a video when the app is restarted? Thaks

**Comment by AndreyPavlenko** [comment]
Sorry, but I didn't get the question. When the playback stops, the video position is saved. When you start the same video again, it starts from the saved position.

**Reply by wovels** [reply]
basically, if you're watching a YouTube video, the EVERY time you turn off your car and turn it back on to continue watching from the same place in YouTube video you need to:
1. Open Fermata Auto
2. Tap Youtube
3. Tap My Library
4. Tap the last one from recently watched videos
5. Finally, you continue watching the video

When Fermata Auto opens there is a progress bar which indicates that it is still on that YouTube video (you can tell it is by the total length of the video shown on the progress bar), but nothing happens when you tap Play/Pause, the button changes from Play to Pause, but the video doesn't resume and there's no sound.
Therefore, to resume playback on Youtube you actually have to go through all those steps I listed. If there's any way the playback from Youtube can resume itself with a simple tap of Play/Pause button - that would be amazing.

**Reply by chavalino** [reply]
Actually, I would like the option to always start playback from the beginning, i.e. not saving the position. Let me tell you why.
I use Fermata to watch music videos and sometimes I'm not on the mood to watch some song so I skip it after some seconds or when the song is ending too slow. But some other day I would like to watch the music video, but it doesn't start at the beginning.
Please add confgurable option: "remember playback position".

**Reply by AndreyPavlenko** [reply]
> Please add confgurable option: "remember playback position".

There is such an option - https://github.com/AndreyPavlenko/Fermata/discussions/167#discussioncomment-2724076

**Reply by AndreyPavlenko** [reply]
> but nothing happens when you tap Play/Pause

It seems like a bug. I'll check.

**Reply by chavalino** [reply]
Not intuitive, but works! Thanks. Appreciated. Sent my donation. Latest version is quite stable.

**Comment by picozzo71** [comment]
sorry for my English.  What you write is true.  I mean when android auto stops (app shutdown).  I wish that when starting YouTube I could have the last video I watched, to be able to continue viewing.
